src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt

```diff
@@ -0,0 +1,43 @@
+package hs.kr.equus.user.global.utils.pass
+
+import hs.kr.equus.user.domain.auth.exception.InvalidOkCertConnectException
+import kcb.module.v3.OkCert
+import kcb.module.v3.exception.OkCertException
+import kcb.org.json.JSONObject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class PassUtil {
+
+    companion object {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line